pandas/core/indexes/datetimes.py

```diff
@@ -2239,6 +2239,27 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     Returns
     -------
     rng : DatetimeIndex
+
+    Examples
+    --------
+    >>> pd1 = pd.date_range(start='1/1/2018', end='2/1/2018')
+    >>> pd1
+    DatetimeIndex(['2018-01-01', '2018-01-02', '2018-01-03', '2018-01-04',
+               '2018-01-05', '2018-01-06', '2018-01-07', '2018-01-08',
```

Comment:
CAn you align all these? Also, you could make it a bit shorter so it's easier to read.

Reply:
Sorry for replying a bit late was there with my friend for some important office work on weekend. I will make changes asap.